### PR TITLE
Moved Deployment creation steps from "Deploy via CLI" to "Configure a Deployment"

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/209

This PR contains very little new content. Rather, it reorganizes existing content by shifting setup from one guide to another. 

I also took this opportunity to remove lettered steps from our guides, as these aren't properly recognized in Markdown as the steps they are. 

If these changes look good, I'll update the Enterprise versions of these guides with the same changes.

### CODE REVIEW

File: cloud/stable/03_deploy/02_configure-deployment.md [hunk 1]
```diff
@@ -6,59 +6,66 @@ description: "How to configure your Airflow Deployment's Resources on Astronomer
 
 ## Overview
 
-Once you've [created an Airflow Deployment](https://www.astronomer.io/docs/cloud/stable/deploy/deploy-cli), you can configure it via the Astronomer UI based on the needs of your organization.
+An Airflow Deployment is an instance of Airflow that you've spun up either via the Astronomer UI or the Astronomer CLI. Under the hood, each Airflow Deployment gets its own Kubernetes namespace and has a reserved set of dedicated resources and an underlying Postgres Metadata Database.
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/02_configure-deployment.md:9]:
```suggestion
An Airflow Deployment on Astronomer is an instance of Apache Airflow that was created either via the Astronomer UI or the Astronomer CLI. Each Airflow Deployment on Astronomer is hosted on a single Kubernetes namespace, has a dedicated set of resources, and operates with an isolated Postgres Metadata Database.
```

File: cloud/stable/03_deploy/02_configure-deployment.md [hunk 2]
```diff
@@ -6,59 +6,66 @@ description: "How to configure your Airflow Deployment's Resources on Astronomer
...
 
-The **Settings** tab of your Airflow Deployment on Astronomer is the best place to modify resources for your Deployment. Specifically, you can:
+This guide walks you through the process of creating and configuring Deployments on Astronomer.
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/02_configure-deployment.md:11]:
```suggestion
This guide walks you through the process of creating and configuring an Airflow Deployment on Astronomer.
```

File: cloud/stable/03_deploy/02_configure-deployment.md [hunk 3]
```diff
@@ -6,59 +6,66 @@ description: "How to configure your Airflow Deployment's Resources on Astronomer
...
-1. Select an Airflow Executor
-2. Allocate resources to your Airflow Scheduler and Webserver
-3. Set Scheduler Count (*Airflow 2.0+ only*)
-4. Set Worker Count (*Celery only*)
-5. Adjust your Worker Termination Grace Period (*Celery only*)
-6. Add Extra Capacity (*Kubernetes only*)
+## Prerequisites
 
-![Astro UI Executor Config](https://assets2.astronomer.io/main/docs/astronomer-ui/v0.23-astro-UI-executor.png)
+To create a Deployment, you'll need:
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/02_configure-deployment.md:15]:
```suggestion
To create an Airflow Deployment, you'll need:
```

File: cloud/stable/03_deploy/02_configure-deployment.md [hunk 4]
```diff
@@ -6,59 +6,66 @@ description: "How to configure your Airflow Deployment's Resources on Astronomer
...
+* An Astronomer [Workspace](https://www.astronomer.io/docs/cloud/stable/deploy/manage-workspaces).
 
-> **Note:** Adjusting resources will affect your monthly bill on Astronomer Cloud. For more information on resource cost and pricing, refer to [Cloud Pricing](https://www.astronomer.io/docs/cloud/stable/resources/pricing).
+## Create a Deployment
 
-## Select an Executor
+To create an Airflow Deployment on Astronomer:
 
-The Airflow [Executor](https://airflow.apache.org/docs/apache-airflow/stable/executor/index.html) works closely with the Airflow Scheduler to decide what resources will complete tasks as they're queued. The difference between Executors comes down to their available resources and how they utilize those resources to distribute work.
+1. Log into [Astronomer Cloud](https://app.gcp0001.us-east4.astronomer.io/), open your Workspace, and click **New Deployment**.
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/02_configure-deployment.md:24]:
```suggestion
1. Log in to [Astronomer Cloud](https://app.gcp0001.us-east4.astronomer.io/), open your Workspace, and click **New Deployment**.
```

File: cloud/stable/03_deploy/02_configure-deployment.md [hunk 5]
```diff
@@ -6,59 +6,66 @@ description: "How to configure your Airflow Deployment's Resources on Astronomer
...
-- [Local Executor](https://airflow.apache.org/docs/apache-airflow/stable/executor/local.html)
-- [Celery Executor](https://airflow.apache.org/docs/apache-airflow/stable/executor/celery.html)
-- [Kubernetes Executor](https://airflow.apache.org/docs/apache-airflow/stable/executor/kubernetes.html)
+  * **Name**
+  * **Description** (Optional)
+  * **Airflow Version**: We recommend using the latest version.
+  * **Executor**: We recommend starting with Local.
 
-Though it largely depends on your use case, we recommend the Local Executor for development environments and the Celery or Kubernetes Executors for production environments operating at scale.
+3. Click **Create Deployment** and give the Deployment a few moments to spin up. Afterwards, you'll have access to your Deployment dashboard:
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/02_configure-deployment.md:33]:
```suggestion
3. Click **Create Deployment** and give the Deployment a few moments to spin up. Within a few seconds, you'll have access to the **Settings** page of your new Deployment:
```

File: cloud/stable/03_deploy/02_configure-deployment.md [hunk 6]
```diff
@@ -6,59 +6,66 @@ description: "How to configure your Airflow Deployment's Resources on Astronomer
...
+![New Deployment Celery Dashboard](https://assets2.astronomer.io/main/docs/deploying-code/v0.23-new_deployment-dashboard.png)
 
-For a detailed breakdown of each Executor, read Astronomer's [Airflow Executors Explained](https://www.astronomer.io/guides/airflow-executors-explained).
+The **Settings** tab of your Airflow Deployment is the best place to modify resources for your Deployment. Specifically, you can:
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/02_configure-deployment.md:36]:
```suggestion
This tab is the best place to modify resources for your Deployment. Specifically, you can:
```

File: cloud/stable/03_deploy/03_deploy-cli.md [hunk 7]
```diff
@@ -22,113 +22,69 @@ For those looking to automate the deploy process, refer to [Deploy to Astronomer
...
+## Step 4: Validate Your Changes
 
 If it's your first time deploying, expect to wait a few minutes for the Docker Image to build.
 
 To confirm that your deploy was successful, navigate to your Deployment in the Astronomer UI and click **Open Airflow** to see your changes in the Airflow UI.
 
-#### What gets Deployed?
+### What gets deployed?
 
 Everything in the project directory where you ran `$ astro dev init` is bundled into a Docker image and deployed to your Airflow Deployment on Astronomer Cloud. We don't deploy any of the metadata associated with your local Airflow environment (e.g. task history, Airflow Connections and Variables set in the Airflow UI, etc.) - just the code.
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/03_deploy-cli.md:83]:
```suggestion
Everything in the project directory where you ran `$ astro dev init` is bundled into a Docker image and deployed to your Airflow Deployment on Astronomer Cloud. Astronomer exclusively deploys the code in your project and does not push any of the metadata associated with your local Airflow environment, including task history and any Airflow Connections or Variables set locally in the Airflow UI.
```

**Review by paolaperaza — COMMENTED** [review]
A few small comments here @jwitz but this looks good 😊

**Comment by jwitz** [comment]
Thanks for the suggestions @paolaperaza , I added all of them to Cloud and then ported over the docs to Enterprise. In addition, I made some wording tweaks to the intros of the Deploy via CLI docs since they still mentioned configuring Deployments.

Working on the PR for all of the link changes/ redirects. Will hopefully have that done soon so that we can merge these at the same time!

File: enterprise/next/04_deploy/03_deploy-cli.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+* [The Astronomer CLI](/docs/enterprise/stable/develop/cli-quickstart/) installed.
+* An Astronomer platform at `app.BASEDOMAIN`.
+* An Astronomer [Workspace](https://www.astronomer.io/docs/enterprise/stable/deploy/manage-workspaces) with at least one active [Airflow Deployment](https://www.astronomer.io/docs/enterprise/stable/deploy/configure-deployment).
+
+## Step 1: Authenticate to Astronomer
+
+To authenticate via the Astronomer CLI, run:
+
+```sh
+$ astro auth login app.BASEDOMAIN
```
Comment (paolaperaza) [inline enterprise/next/04_deploy/03_deploy-cli.md:30]:
```suggestion
$ astro auth login BASEDOMAIN
```
No `app` here! Source of confusion for a lot of users 🤦‍♀️

File: enterprise/next/04_deploy/03_deploy-cli.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+## Step 2: Confirm Your Workspace and Deployment
+
+From the Astronomer CLI, you're free to push code to any Airflow Deployment you have access to as long as you have the appropriate deployment-level permissions to do so.
```
Comment (paolaperaza) [inline enterprise/next/04_deploy/03_deploy-cli.md:35]:
```suggestion
From the Astronomer CLI, you're free to push code to any Airflow Deployment you have access to as long as you have the appropriate deployment-level permissions to do so. For more information on both Workspace and Deployment-level permissions on Astronomer, refer to [User Permissions](https://www.astronomer.io/docs/enterprise/v0.23/manage-astronomer/workspace-permissions).
```

File: enterprise/v0.23/04_deploy/03_deploy-cli.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+* [The Astronomer CLI](/docs/enterprise/v0.23/develop/cli-quickstart/) installed.
+* An Astronomer platform at `app.BASEDOMAIN`.
+* An Astronomer [Workspace](https://www.astronomer.io/docs/enterprise/v0.23/deploy/manage-workspaces) with at least one active [Airflow Deployment](https://www.astronomer.io/docs/enterprise/v0.23/deploy/configure-deployment).
+
+## Step 1: Authenticate to Astronomer
+
+To authenticate via the Astronomer CLI, run:
+
+```sh
+$ astro auth login app.BASEDOMAIN
```
Comment (paolaperaza) [inline enterprise/v0.23/04_deploy/03_deploy-cli.md:30]:
```suggestion
$ astro auth login BASEDOMAIN
```

File: enterprise/v0.23/04_deploy/03_deploy-cli.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+## Step 2: Confirm Your Workspace and Deployment
+
+From the Astronomer CLI, you're free to push code to any Airflow Deployment you have access to as long as you have the appropriate deployment-level permissions to do so.
```
Comment (paolaperaza) [inline enterprise/v0.23/04_deploy/03_deploy-cli.md:35]:
```suggestion
From the Astronomer CLI, you're free to push code to any Airflow Deployment you have access to as long as you have the appropriate deployment-level permissions to do so. For more information on both Workspace and Deployment-level permissions on Astronomer, refer to [User Permissions](https://www.astronomer.io/docs/enterprise/v0.23/manage-astronomer/workspace-permissions).
```

**Review by paolaperaza — APPROVED** [review]
Awesome, looks great @jwitz ! Two super small final suggestions, but approving preemptively.

Can you also pass these changes to Enterprise v0.16?

**Comment by jwitz** [comment]
For sure, just added to v0.16. Merging now!

**Pull request merged**